frontend/io-webapp/src/reducer.js

```diff
@@ -18,27 +18,37 @@ export const initialState = {
   questionWidth: "100%",
   sessionOwner: "",
   participants: [],
+  isStatsVisible: false,
 };
 
 export const reducer = (state, action) => {
   switch (action.type) {
-    case "SET_STAGE_START": console.log(state);
+    case "SET_STAGE_START":
+      console.log(state);
```

Comment:
To loggowanie jest do debugowania jakiegoś, prawda? Bo rozumiem, że docelowo to tutaj nie jest potrzebne, prawda?

Reply:
Tak, ten kod do debugowania zazwyczaj usuwam za każdym razem, gdy mi wpadnie w oko, tego akurat nie zauważyłem.